Task8/Task8/Program.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.Remoting.Metadata.W3cXsd2001;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task8
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите x");
+            string x = Console.ReadLine();
+            double y = double.Parse(x);
+            Console.WriteLine(Function(y));
+            Console.ReadKey();
+        }
+
+        static double Function(double x)
+        {
+            if (x > 2)
+                return (2);
+            else if ((0 < x) && (x<=2))
+            {
+        
+                    return 0;
+            }
+            else
+                return (-3 * x);
+
+
+        }
+        
+    }
```

Comment:
1. Не используйте ненужных пустых строк и лишние скобки. Это затрудняет чтение кода.
2. Если выполнение операторов внутри конструкции `if` заканчивается оператором return, то  `else` дальше уже не нужен.
3. Если поменять порядок проверки, то не придется проверять двойное неравенство.
```suggestion
        static double Function(double x)
        {
            if (x > 2)
                return 2;
            
            if (x <= 0)
                return -3 * x;

            return 0;
        }        
    }
```